src/test/e2e/helpers/test-helper.ts

```diff
@@ -45,15 +55,29 @@ function httpApis(
 
     return {
         createFeature: (
-            name: string,
+            feature: string | CreateFeatureSchema,
             project: string = DEFAULT_PROJECT,
             expectedResponseCode: number = 201,
         ) => {
+            let body = feature;
+            if (typeof feature === 'string') {
+                body = {
+                    name: feature,
+                };
+            }
             return request
                 .post(`${base}/api/admin/projects/${project}/features`)
-                .send({
-                    name,
-                })
+                .send(body)
+                .set('Content-Type', 'application/json')
+                .expect(expectedResponseCode);
+        },
+
+        getFeatures(
+            name?: string,
+            expectedResponseCode: number = 200,
+        ): supertest.Test {
+            return request
+                .get(`/api/admin/features${name ? '/' + name : ''}`)
```

Comment:
This follows REST conventions of having the feature resource and being able to get a single one of them. Alternatively, we could have two methods, but again, because this is test code, I thought it was better to simplify